src/nm-manager.c

```diff
@@ -4273,7 +4275,12 @@ _internal_activate_device (NMManager *self, NMActiveConnection *active, GError *
 	/* Check slaves for master connection and possibly activate them */
 	autoconnect_slaves (self, connection, device, nm_active_connection_get_subject (active));
 
-	{
+	multi_connect = _nm_connection_get_multi_connect (NM_CONNECTION (connection));
+	if (   multi_connect == NM_CONNECTION_MULTI_CONNECT_MULTIPLE
+	    || (   multi_connect == NM_CONNECTION_MULTI_CONNECT_MANUAL_MULTIPLE
+	        && nm_active_connection_get_activation_reason (active) != NM_ACTIVATION_REASON_AUTOCONNECT)) {
```

Comment:
Perhaps it would be better to explicitly check that the reason is USER_REQUEST here?

Should MANUAL_MULTIPLE allow to activate, for example, connections with reason EXTERNAL, ASSUME and AUTOCONNECT_SLAVES?